python/tests/xir/test_program.py

```diff
@@ -279,24 +275,24 @@ def test_add_called_function(self, program):
 
     def test_add_declaration(self, program):
         """Tests that declarations can be added to an XIR program."""
-        tan = FuncDeclaration("tan", 1)
+        tan = Declaration("tan", ["x"], (), declaration_type="function")
```

Comment:
Sorry about these!

Reply:
No worries! It's more due to my not-so-great declaration classes that I had implemented earlier. 
😉